server/e2e/integration_schema_test.go

```diff
@@ -131,6 +131,115 @@ func TestIntegrationScemaFilterAPI(t *testing.T) {
 	obj1.Value("lastModified").NotNull()
 }
 
+// GET /projects/{projectIdOrAlias}/schemata with sorting
+func TestIntegrationSchemaFilterSorting(t *testing.T) {
+	endpoint := "/api/projects/{projectIdOrAlias}/schemata"
+	e := StartServer(t, &app.Config{}, true, baseSeeder)
+
+	// createdAt and ascending
+	res1 := e.GET(endpoint, pid).
+		WithHeader("authorization", "Bearer "+secret).
+		WithQuery("page", 1).
+		WithQuery("perPage", 10).
+		WithQuery("sort", "createdAt").
+		WithQuery("dir", "asc").
+		Expect()
+	res1.Status(http.StatusOK)
+	models1 := res1.JSON().
+		Object().
+		Value("models").
+		Array()
+
+	models1.Value(0).Object().Value("id").String().IsEqual(mId0.String())
+	models1.Value(1).Object().Value("id").String().IsEqual(mId1.String())
+	models1.Value(2).Object().Value("id").String().IsEqual(mId2.String())
+	models1.Value(3).Object().Value("id").String().IsEqual(mId3.String())
+	models1.Value(4).Object().Value("id").String().IsEqual(mId4.String())
+	models1.Value(5).Object().Value("id").String().IsEqual(mId5.String())
+	models1.Value(6).Object().Value("id").String().IsEqual(dvmId.String())
```

Comment:
```suggestion
	res.JSON().Path("$.models[:].id").Array().IsEqual([]string{mId0.String(), /*...*/})
```
json path gives more readable code